### DISCUSSION TITLE
New Version Marketplace

### DISCUSSION THREAD

**Discussion by Daniel127** [StanDeMan/RaspberryDebugger#12] [Q&A] [answered]
Hello @StanDeMan.

When can we have a new version published in the Visual Studio Marketplace?.

What things are still to be improved or to be done?

**Comment by StanDeMan** [comment, marked as answer]
Hi @Daniel127 
Have an issues starting the very first time with debugging. WebBrowser gets opened but the page is not displayed.  
This is only the very first time - after starting Visual Studio and only using Kestrel.
Do you have this issue too?
The best I can do is list this as known issue and publish a new version on Marketplace.

**Reply by Daniel127** [reply]
Hello,

I have not had that problem, sometimes raspberrypi.local:port does not work but I change to raspberrypi:port and it works perfect, I understand that this is already a networking problem and not of the Debugger.
